app/models/manageiq/providers/hawkular/inventory/persister/availability_updates.rb

```diff
@@ -1,40 +1,44 @@
 module ManageIQ::Providers
   class Hawkular::Inventory::Persister::AvailabilityUpdates < Hawkular::Inventory::Persister::MiddlewareManager
     def self.save_deployments(ems, collection)
-      ::ActiveRecord::Base.transaction do
```

Comment:
I think the transaction should remain so that deployments are persisted atomically.

Reply:
Still in the PR, [here](https://github.com/ManageIQ/manageiq-providers-hawkular/pull/67/files#diff-a41892d210b721576656574d8d9d0846R26)